### PR TITLE
Change DocsTopic to AssetGroup and HeadlessCMS to Rafter - kyma repo

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Changed DocsTopic to AssetGroup and HeadlessCMS to Rafter

**Related issue(s)**
See also /kyma-project/rafter/issues/10

### CODE REVIEW

File: docs/kyma/05-03-overrides.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ cat <<EOF | kubectl apply -f -
 apiVersion: v1
 kind: ConfigMap
 metadata:
```
Comment (kazydek) [inline docs/kyma/05-03-overrides.md]: In this doc, there is still one asset-tore reference:
```
The Installer uses an `asset-store` top-level chart that contains a template with the following value reference:
```

File: docs/README.md [hunk 2]
```diff
@@ -25,7 +24,7 @@ Start with the overarching [Kyma](./kyma) documentation where you can find the g
 
 Read it directly in GitHub or inside the Kyma Console if you have access to the Kyma cluster.
 
-The navigation order of the documentation page is based on the **cms.kyma-project.io/order** label specified in the [ClusterDocsTopic](../resources/core/charts/docs/charts/content-ui/templates) for a given Kyma component.
+The navigation order of the documentation page is based on the **rafter.kyma-project.io/order** label specified in the [ClusterAssetGroup](../resources/core/charts/docs/charts/content-ui/templates) for a given Kyma component.
```
Comment (magicmatatjahu) [inline docs/README.md]: Path will change:

```md
The navigation order of the documentation page is based on the **rafter.kyma-project.io/order** label specified in the [ClusterAssetGroup](../resources/core/charts/content-ui/templates) for a given Kyma component.
```

File: docs/console/03-04-rafter-in-console.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+>**NOTE:** OpenAPI, OData, and AsyncAPI specifications rendered in the Console UI follow the [Fiori 3 Fundamentals](https://sap.github.io/fundamental/) styling standards.
+
+The source files are uploaded directly to the given storage without any modifications, except for the following source types:
+
+- `asyncapi` that the AsyncAPI Service validates and, if required, converts to version 2.0 and the `json` format.
+- `markdown` from which the Front Matter Service extracts front matter metadata.
+
+![Specification types](./assets/spec-types.svg)
+
+> **TIP:** The default Kyma webhooks that convert and validate `asyncapi` source files and extract metadata from `markdown` files are defined in the [`webhook-config-map.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/rafter/charts/rafter-controller-manager/templates/webhook-config-map.yaml) ConfigMap.
```
Comment (magicmatatjahu) [inline docs/console/03-04-rafter-in-console.md]: Path will change:

```markdown
> **TIP:** The default Kyma webhooks that convert and validate `asyncapi` source files and extract metadata from `markdown` files are defined in the [`webhook-config-map.yaml`](https://github.com/kyma-project/kyma/blob/master/resources/rafter/charts/controller-manager/templates/webhook-config-map.yaml) ConfigMap.
```

File: docs/console/03-04-rafter-in-console.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+<!-- The link from the TIP may change. Check once Rafter is already in Kyma. -->
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply docs/console/03-04-rafter-in-console.md]: You can remove comment :)

File: docs/kyma/05-03-overrides.md [hunk 5]
```diff
@@ -70,15 +70,15 @@ cat <<EOF | kubectl apply -f -
...
   namespace: kyma-installer
   labels:
     installer: overrides
-    component: assetstore
+    component: rafter
     kyma-project.io/installation: ""
 data:
-  minio.resources.limits.memory: 512Mi #increased from 128Mi
-  minio.resources.limits.cpu: 250m #increased from 100m
+  rafter-controller-manager.minio.resources.limits.memory: 512Mi #increased from 128Mi
```
Comment (magicmatatjahu) [inline docs/kyma/05-03-overrides.md]: Only `controller-manager` prefix :)

File: docs/kyma/05-03-overrides.md [hunk 6]
```diff
@@ -70,15 +70,15 @@ cat <<EOF | kubectl apply -f -
...
+  rafter-controller-manager.minio.resources.limits.cpu: 250m #increased from 100m
```
Comment (magicmatatjahu) [inline docs/kyma/05-03-overrides.md]: Only `controller-manager` prefix :)

File: docs/README.md (hunk 2 above)
Comment (magicmatatjahu) [reply docs/README.md]: Sorry, but path `../resources/core/charts/docs/charts/content-ui/templates`

File: docs/console/03-04-rafter-in-console.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply docs/console/03-04-rafter-in-console.md]: Correct link `https://github.com/kyma-project/kyma/blob/master/resources/rafter/charts/controller-manager/templates/webhook-config-map.yaml`

[the remark this answers is not in the document]
Comment (mmitoraj) [reply docs/console/03-04-rafter-in-console.md]: so nothing has changed... ;)

**Pull request merged**